site/en/blog/floc/index.md

```diff
@@ -66,21 +72,51 @@ The cohort data made available looks like this:
 ``` json
     {
         "id": "14159",
-        "version": "chrome.1.0"
+        "version": "chrome.2.1"
     }
```

Comment:
you've got extra whitespace here:

![image](https://user-images.githubusercontent.com/1066253/115048885-f05b6c00-9e8e-11eb-936d-9a9bb2faf409.png)

Reply:
Nice catch! Thank you.